contrib/images/Makefile

```diff
@@ -30,7 +30,10 @@ cosmos-relayer-rmi:
 	docker rmi babylonlabs-io/cosmos-relayer 2>/dev/null; true
 
 build-ibcsim-bcd:
-	docker build --tag babylonlabs-io/ibcsim-bcd -f ibcsim-bcd/Dockerfile ${BABYLON_FULL_PATH}
+	cp -a $(BABYLON_FULL_PATH)/babylon-sdk/ $(BABYLON_FULL_PATH)/contrib/images/ibcsim-bcd
```

Comment:
Wdyt we use `git clone` for getting babylon-sdk repo to a `gitignore`-ed directory, instead of a submodule? suggesting this as submodules are quite troublesome due to those `git submodule init` and `git submodule update` stuff

Reply:
Yes, this is one of the concerns I mentioned above. In my opinion this entire schema is wrong, either with git submodules or with cloning. We should move to a version in which each repository builds and publishes one (or a number) of docker images for its related services. Then the other repos pull this tagged image from docker hub, or another docker registry.

For now, I'll change to cloning the repo and checking out a version tag, and we can address this proposal above at some point, as part or as a follow up of (refined) https://github.com/babylonlabs-io/pm/issues/118.

Reply:
Sounds good :+1: